### PR TITLE
CAL1-4 Facility Profile View: Capacity

### PR DESCRIPTION
**Pull request by Nasir2712**
#### Which User story does this relate to?
- https://osi-cwds.atlassian.net/browse/CAL1-4

#### Brief feature description
-  The capacity_last_changed is going to be the new field which has not been added yet by the api. (Will update the description once they send us this field)

#### Does this PR block/effect other stories?
- no

#### Do any commands needs to be run in order to run the project with this PR?
- n/A

#### Code coverage percent diff as a result of this PR?
-

### CODE REVIEW

File: app/javascript/search/common/commonUtils.js [hunk 1]
```diff
@@ -24,6 +24,10 @@ export const phoneNumberOrNa = (phone) => {
   }
 }
 
+export const checkforDateOrNa = (date) => {
+  return (date === null || date === undefined) ? 'N/A' : date.split(' ')[0]
```
Comment (ratneshraval) [inline app/javascript/search/common/commonUtils.js]: Should do a length check too? Maybe an improvement of the following?
```
if (Boolean(date) && (date.split(' ').length > 1) )
{
  date.split(' ')[0]
} else {
  'N/A'
}
```

Comment (Nasir2712) [reply app/javascript/search/common/commonUtils.js]: nice i like it updating it.

File: test/javascript/facility/address_tests.js [hunk 2]
```diff
@@ -27,7 +27,7 @@ describe('Verify Facility Address', function () {
       'capacity': 10,
       'license_effective_date': '2012-10-01',
       'original_application_recieved_date': '2012-07-18',
-      'last_visit_date': '2017-04-14 00:00:00',
+      'last_visit_date': '2017-04-14',
```
Comment (ratneshraval) [inline test/javascript/facility/address_tests.js]: No test for undefined/null date?

Comment (Nasir2712) [reply test/javascript/facility/address_tests.js]: added test cases for null and value

**Pull request merged**